tests/crypto_policy_test.go

```diff
@@ -109,9 +111,15 @@ var _ = Describe("Crypto Policy", func() {
 	Context("setting Crypto Policy", func() {
 		DescribeTable("Adhere to defined TLSConfig", func(tlsConfigTestPermutation tlsConfigTestPermutation) {
 			pod := operatorPod()
+			validatorPod := templateValidatorPod()
+
 			applyTLSConfig(tlsConfigTestPermutation.openshiftTLSPolicy)
-			testMetricsEndpoint(pod, tlsConfigTestPermutation)
-			testWebhookEndpoint(pod, tlsConfigTestPermutation)
+			Expect(testMetricsEndpoint(pod, tlsConfigTestPermutation)).To(Succeed())
+			Expect(testWebhookEndpoint(pod, tlsConfigTestPermutation)).To(Succeed())
+
+			Eventually(func() error {
+				return testValidatorEndpoint(validatorPod, tlsConfigTestPermutation)
+			}, env.ShortTimeout(), time.Second).Should(Succeed())
```

Comment:
Why do other endpoints not need an eventually?

Reply:
Because `applyTLSConfig()` waits until the new configuration is read by SSP. SSP needs to go to `Deploying` state and then back to `Deployed`.
We don't have any way to check if template-validator reloaded the configuration.